### ISSUE TITLE
无效节点太多了

### ISSUE THREAD

**Issue by itnewgay** [bannedbook/fanqiang#586]
本项目提供多个软件和多项服务，请明确说明您反馈的问题是有关哪一个软件，或哪一个服务？
V2ray 的速度很快.  但是里面的订阅节点里.已经不能用的节点太多了.希望能够优化维护一下.把无效的节点拿掉.

**Comment by bannedbook** [comment]
是说 v2rayb 吗，需要自己来 执行一个 批量测试真连接延迟 ，然后删除无效即可

**Comment by itnewgay** [comment]
> 是说 v2rayb 吗，需要自己来 执行一个 批量测试真连接延迟 ，然后删除无效即可

是的. 我知道 你的方法 ,但是这个太麻烦了. 现在无效节点越来越多了.每次删除后,再更新订阅.又出来了. 希望能直接在订阅里直接 就把已经不能使用的节点删除掉.  另外:感谢你们!    

你们的安卓版本内置的节点最好用. 但是好像WINDOWS版本的没办法获取到那些节点。

**Comment by bannedbook** [comment]
这个暂且做不到，你要嫌麻烦，可以试试clash通道，clash会自动校验

**Comment by Alvin9999** [comment]
> > 是说 v2rayb 吗，需要自己来 执行一个 批量测试真连接延迟 ，然后删除无效即可
> 
> 是的. 我知道 你的方法 ,但是这个太麻烦了. 现在无效节点越来越多了.每次删除后,再更新订阅.又出来了. 希望能直接在订阅里直接 就把已经不能使用的节点删除掉. 另外:感谢你们!
> 
> 你们的安卓版本内置的节点最好用. 但是好像WINDOWS版本的没办法获取到那些节点。

windows电脑可以试试最新的免费v2ray账号，速度有提升。
